program-runtime/src/compute_budget.rs

```diff
@@ -13,6 +13,7 @@ use {
 pub const DEFAULT_INSTRUCTION_COMPUTE_UNIT_LIMIT: u32 = 200_000;
 pub const MAX_COMPUTE_UNIT_LIMIT: u32 = 1_400_000;
 const MAX_HEAP_FRAME_BYTES: u32 = 256 * 1024;
+pub const MAX_ACCOUNTS_DATA_SIZE_LIMIT: u32 = 100_000_000;
```

Comment:
Not sure if 100MB is proper for max accounts data a transaction is allowed to load. It roughly translates to 71 bytes/CU (with 1.4mil CU per transaction), which is cheaper than cost_model's 5 bytes/cu [rate](https://github.com/solana-labs/solana/blob/master/runtime/src/block_cost_limits.rs#L30). But I think @Lichtso is using 250 byte/CU in runtime?

Reply:
commit 908e7d5e1b9a1a5726c32ca53161598e2815c905 set default to 10MB, and Max to 100Mb, per transaction. Also added `getter`s to allow these values to be changed with version / feature.